migrate/20190305124422_create_story_map_links.rb

```diff
@@ -0,0 +1,11 @@
+class CreateStoryMapLinks < ActiveRecord::Migration
+  def change
+    create_table :story_map_links do |t|
+      t.integer :wdpa_id
```

Comment:
Referring to the comment I made on the other PR, I think it would be better to use `t.references :protected_area` to create the key that handles the relationship